cmd/gateway-main.go

```diff
@@ -216,7 +215,7 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 	globalObjLayerMutex.Unlock()
 
 	// Prints the formatted startup message once object layer is initialized.
-	if !quietFlag {
+	if quietMode {
```

Comment:
@balamurugana The update text should not be shown even if json flag is set.

Reply:
Then you would need to fix logger or place which doesn't handle json and quiet flag properly.  This change is not appropriate

Reply:
I don't understand the comment. If json flag or quiet flag is set, we do not display startup messages and this update message also should not be displayed if json or quiet flags are enabled. I can change the code such that I move the checking for flags into a function and check if quietmode is enabled in logger.